robottelo/common/helpers.py

```diff
@@ -177,16 +179,24 @@ def invalid_names_list():
     List of invalid names for input testing.
     """
     return [
-        gen_string("alpha", 300),
-        gen_string("numeric", 300),
-        gen_string("alphanumeric", 300),
-        gen_string("utf8", 300),
-        gen_string("latin1", 300),
-        gen_string("html", 300),
-        gen_string("alpha", 256)
+        gen_string('alphanumeric', 300),
+        gen_string('alpha', 300),
+        gen_string('cjk', 300),
+        gen_string('latin1', 300),
+        gen_string('numeric', 300),
+        gen_string('utf8', 300),
+        gen_string('html', 300),
     ]
```

Comment:
Nitpick: Couldn't we specify the max lengths as variables for an easier manipulation next time?